file_paths.md

```diff
@@ -163,160 +163,201 @@ The package will create the following services `puppet`, `mcollective`,
+            curl.exe *
+            curl-config *
+            daemon.bat *
+            daemon.rb *                           # windows service daemon
+            elevate.exe *                         # Used to elevate interactive commands
+            elevate.exe.config *
+            environment.bat *                     # setup LOAD_PATH, globally used by all our .bat files, to be symlinked to C:\Program Files\Puppet Labs\bin
+            erb.bat *
+            extlookup2hiera *
+            extlookup2hiera.bat *
+            facter.bat *                          # bat file wrapper
             facter.exe *
+            facter_interactive.bat *
+            gem.bat *
+            hiera *                               # ruby bin wrapper
+            hiera.bat *                           # bat file wrapper
+            irb.bat *
+            libcpp-pcp-client.so *
+            libcurl-4.dll *                       # curl dll
+            libeay32.dll *                        # openssl dll
             libfacter.so *
-            lib*.dll *                        # gcc, mingw, boost libraries
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
+            libgcc_s_seh-1.dll *
+            libstdc++-6.dll *
+            libwinpthread-1.dll *
+            mco *                                 # ruby bin wrapper
+            mco.bat *                             # bat file wrapper
+            mco_daemon.bat *
+            mcollectived *                        # ruby bin wrapper
+            minitar *
+            minitar.bat *
+            nssm.exe *                            # NSSM used to run pxp-agent
+            openssl.exe *
+            puppet *                              # ruby bin wrapper
+            puppet.bat *                          # bat file wrapper
+            puppet_interactive.bat *
+            puppet_shell.bat *                    # targets for shortcuts
+            puppetres.dll *                       # event log message resource dll
+            pxp-agent.exe *
+            rake.bat *
+            rdoc.bat *
+            ri.bat *
+            ruby.exe *
+            rubyw.exe *
+            run_facter_interactive.bat *
+            run_puppet_interactive.bat *
+            ssleay32.dll *                        # openssl dll
+            stompcat *
+            stompcat.bat *
+            testrb.bat *
+            x64-msvcrt-ruby210.dll *              # ruby dll
+            zlib1.dll *
 
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
+        include *
+            boost *
```

Comment:
Ok, I misunderstood some of the documentation. By
```
lib*.dll *                        # gcc, mingw, boost libraries
```
we meant libboost_nowide.dll. We no longer ship that, so it could be dropped from the comment. And we do ship `include/boost/nowide`, which is where boost in includes comes from. So no changes needed here.